django/contrib/auth/hashers.py

```diff
@@ -211,6 +221,11 @@ def safe_summary(self, encoded):
         raise NotImplementedError('subclasses of BasePasswordHasher must provide a safe_summary() method')
 
     def must_update(self, encoded):
+        decoded = self.decode(encoded)
+        if 'salt' not in decoded:
+            warnings.warn(f'Decode not fully implemented for {type(self).__name__}.\
```

Comment:
I would put "decode" into quotes or so (@felixxm can certainly tell us what the proper syntax is)

Reply:
You mean as `f'"Decode" not fully ...` ?

Reply:
Not sure what the best option is, maybe 
```
f'Method `decode`not fully…`
```
lets see if someone else chimes in.

Reply:
Cool. I'll give it a bit for someone else to chime in, but I've pushed a commit with your suggestion for the time being.